dockers/docker-sonic-telemetry/telemetry.sh

```diff
@@ -2,36 +2,38 @@
 
 # Try to read telemetry and certs config from ConfigDB.
 # Use default value if no valid config exists
-X509=`sonic-cfggen -d -v "DEVICE_METADATA['x509']"`
-gnmi=`sonic-cfggen -d -v "TELEMETRY['gnmi']"`
-certs=`sonic-cfggen -d -v "TELEMETRY['certs']"`
+TELEMETRY_VARS=`sonic-cfggen -d -t telemetry_vars.j2`
```

Comment:
It will be much easier if you rewrite the whole file by python.

Reply:
the python will carry heavy weight which we are trying to avoid. This could come as a separate PR where Python overhead could be accurately measured.